pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -95,6 +96,7 @@ func Register(ctx context.Context, wContext *wrangler.Context, mgmtCtx *config.M
 		appClient:            mgmtCtx.Project.Apps(""),
 		nsClient:             mgmtCtx.Core.Namespaces(""),
 		clusterClient:        wContext.Mgmt.Cluster(),
+		catalogManager:       mgmtCtx.CatalogManager,
```

Comment:
This wasn't added in the original version: https://github.com/rancher/rancher/pull/29559/files#diff-614af3f88d287b6c050875a460d2ee9a7f4f11877d9d129b9bacfd077832ceddR99

My guess is that one is incorrect not this one.

Reply:
@cbron we talked about this in standup, that is the panic that was found by @thedadams and merge into master separately. I just threw it in here since there is little point to do them separately. I accidentally put it under issue but that second PR is linked.

Reply:
Gotcha.